src/test/groovy/graphql/parser/IDLParserTest.groovy

```diff
@@ -582,5 +588,162 @@ input Gun {
         (typeDef2.getImplements()[1] as TypeName).getName() == 'Baz'
     }
 
+    def "basic type extensions"() {
```

Comment:
how about splitting it up in several tests?

Reply:
ok'